### PR TITLE
Gerg and gross test

### PR DESCRIPTION
**Pull request by royvegard**
Tests for the C++ version of Gerg2008 and Gross.

### CODE REVIEW

File: AGA8CODE/C/Gross_test_01.cpp [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+    GrossHv(x,xGrs,HN,HCH);
+    DensityGross(T,P,xGrs,HCH,D,ierr,herr);
+    PressureGross(T,D,xGrs,HCH,pp,Z,ierr,herr);
+    printf("Outputs-----\n");
+    if (fabs(mm_reference - mm) > 1.0e-8)
+    {
+        printf("Molar mass [g/mol]:                 %0.16g != %0.16g\n",
+            mm_reference, mm);
+        return_value = 1;
```
Comment (ianhbell) [inline AGA8CODE/C/Gross_test_01.cpp]: I would use the ``EXIT_FAILURE`` macro here

File: AGA8CODE/C/Gross_test_01.cpp [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    x.insert(x.begin(), 0.0);
+    double mm = 0;
+    MolarMassGross(x, mm);
+
+    int ierr = 0;
+    std::string herr;
+    double T = 300, P = 10000, Td = 273.15, Th = 298.15, Pd = 101.325;
+    double D = 6.35826, pp = -1, Hv = -1, Hv2 = -1, Gr, HN, HCH, Z=-1;
+
+    int return_value = 0;
```
Comment (ianhbell) [inline AGA8CODE/C/Gross_test_01.cpp]: I would use ``EXIT_SUCCESS`` here instead of 0

File: AGA8CODE/C/GERG2008_test_01.cpp [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+    int ierr = 0;
+    std::string herr;
+    double T = 400, P = 50000, D = 6.36570, Z = 0;
+    double dPdD, d2PdD2, d2PdTD, dPdT, U, H, S, Cv, Cp, W, G, JT, Kappa, A;
+    DensityGERG(0, T, P, x, D, ierr, herr);
+    PropertiesGERG(T, D, x, P, Z, dPdD, d2PdD2, d2PdTD, dPdT, U, H, S,
+        Cv, Cp, W, G, JT, Kappa, A);
+
+    int return_value = 0;
```
Comment (ianhbell) [inline AGA8CODE/C/GERG2008_test_01.cpp]: I would use ``EXIT_SUCCESS`` here instead of 0

File: AGA8CODE/C/GERG2008_test_01.cpp [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+    printf("Inputs-----\n");
+    printf("Temperature [K]:                    400.0000000000000 != %0.16g\n", T);
+    printf("Pressure [kPa]:                     50000.00000000000 != %0.16g\n", P);
+    printf("Outputs-----\n");
+    if (fabs(mm_reference - mm) > 1.0e-8)
+    {
+        printf("Molar mass [g/mol]:                 %0.16g != %0.16g\n",
+            mm_reference, mm);
+        return_value = 1;
```
Comment (ianhbell) [inline AGA8CODE/C/GERG2008_test_01.cpp]: I would use ``EXIT_FAILURE`` here instead of 1

File: AGA8CODE/C/Gross_test_01.cpp (hunk 1 above)
Comment (royvegard) [reply AGA8CODE/C/Gross_test_01.cpp]: Integrated

**Comment by royvegard** [comment]
I implemented your comments for the Aga8 test too.

**Comment by ianhbell** [comment]
Thanks!

**Pull request merged**